aspnet/mvc/views/partial/sample/global.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "projects": [ "src", "test" ],
+  "sdk": {
+    "version": "1.0.0-preview1-002702"
+  }
+}
```

Comment:
Can probably go, but again, created automatically by VS (and I'd rather not have to fight with it every time I re-open the solution).